src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -178,6 +181,14 @@ def _merge_new_units(self, target_path, corpus_dir, new_corpus_dir,
 
     stat_overrides['new_units_added'] = new_units_added
 
+    # Record the stats to make them easily searchable in stackdriver.
+    if new_units_added:
+      logs.log(
+          'New units added to corpus: %d.' % new_units_added,
+          stats=stat_overrides)
```

Comment:
Not that important, but are new_units_added one of the `stat_overrides` kind of redundant. Also unclear why stats should be reported in this log and not elsewhere (I guess we need to do it somewhere arbitrary).

Reply:
I think this logging was added for quick debugging, as we have the stats in the BigQuery otherwise.

Reply:
yes i think stats logging can be moved to its own logs.log call
logs.log('Calculated stats.', stats=stat_overrides).

Reply:
Done!